lib/lndclient/LndClient.ts

```diff
@@ -48,6 +50,7 @@ class LndClient extends BaseClient {
   private invoiceSubscription?: ClientReadableStream<lndrpc.InvoiceSubscription>;
   private reconnectionTimer?: NodeJS.Timer;
   private identityPubKey?: string;
+  private cltvDelta!: number;
```

Comment:
Minor thing but you can assign this a default value here `cltvDelta = 0` and drop the `!` operator and the assignment in the constructor. Fwiw I still think it'd be better just to have this be nullable particularly if we need truthy checks where it's used to prevent divide by zero errors.

Reply:
ok